specification/signalr/resource-manager/Microsoft.SignalRService/stable/2018-10-01/signalr.json

```diff
@@ -319,6 +322,9 @@
           }
         },
         "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
```

Comment:
@juniwang any reason you are adding this to PUT operations? I was under the impression that this is only for POST.

Reply:
Telling from [this doc](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation-options): **When x-ms-long-running-operation is specified, there should also be a x-ms-long-running-operation-options specified.**. 

Will remove it.

Reply:
Removed